www/wp-config.php

```diff
@@ -0,0 +1,83 @@
+<?php
+// ===================================================
+// Load database info and local development parameters
+// ===================================================
+if ( file_exists( dirname( __FILE__ ) . '/local-config.php' ) ) {
```

Comment:
This maybe isn't needed if we're getting details from Consul (which can have different details for each environment)